src/processInput.py

```diff
@@ -69,7 +68,6 @@ def usage():
             if os.path.isfile(filePath):
                 os.remove(filePath)
         print('Done! Removed %d files ' % len(stateFiles.getAllStateFiles()))
-        sys.exit(0)
```

Comment:
why is this gone? i think clean mode was specifically supposed to run separately and then exit early -- its not supposed to continue